test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -194,25 +194,47 @@ func schedulePods(config *testConfig) int32 {
 	config.podCreator.CreatePods()
 
 	prev := 0
+	// On startup there may be a latent period where NO scheduling occurs (qps = 0).
+	// We are interested in low scheduling rates (i.e. qps=2),
 	minQps := int32(math.MaxInt32)
 	start := time.Now()
+
+	// Bake in time for the first pod scheduling event.
+	for {
+		time.Sleep(50 * time.Millisecond)
+		scheduled := config.schedulerConfigFactory.ScheduledPodLister.Indexer.List()
+		// 30,000 pods -> wait till @ least 300 are scheduled to start measuring.
+		// TODO Find out why sometimes there may be scheduling blips in the beggining.
+		if len(scheduled) > config.numPods/100 {
```

Comment:
So here is my take on some of this... I think the algorithm for evaluation here is flawed.   I think we should bin, throw out the high and low and take the average, as lumps should be expected.  This jiggery is far to brittle for my liking.  Standard pattern should be to normalize the curves and eliminate the outliers to determine base level drift on the average distribution.